react/modules/windowListener.ts

```diff
@@ -0,0 +1,59 @@
+import throttle from 'throttleit'
```

Comment:
This module is probably too much for this single use case? Just wanted a way to have "singleton" window listeners of whatever type instead of creating ones for each sticky layout.